tests/unit/cluster-cli.tcl

```diff
@@ -42,9 +26,9 @@ start_multiple_servers 3 [list overrides $base_conf] {
                            127.0.0.1:[srv -2 port]
 
         wait_for_condition 1000 50 {
-            [csi 0 cluster_state] eq {ok} &&
-            [csi -1 cluster_state] eq {ok} &&
-            [csi -2 cluster_state] eq {ok}
+            [CI 0 cluster_state] eq {ok} &&
```

Comment:
i didn't like the use of these upper case functions (considering that for the non-clustered tests we have `proc s` and `proc r`)
i thought maybe for cluster we'll prefix them with `c` and have `proc cs` (or `proc csi`) instead of `CI`, and `cr` instead of `R`.

considering that you'll be porting / migrating existing tests, keeping `R` and `CI` would mean you can in theory copy them as is, and the blame log stays intact (rename + minor changes), but since you changed indentation, git didn't consider these as renames, and instead considered that as new files, so maybe we can use this opportunity to do some terminology refactoring?

Reply:
actually, maybe we don't need `R` at all, and we can use the normal `r`?
i.e. as for `csi` the reason it's different is because it uses a different command (CLISTER INFO instead of INFO).
whereas R and r are just different interfaces for the same thing.

Reply:
My initial thought about this was that I wanted to get everything ported over first, and so it was just useful to have both the two client methods lying around even thought they were very similar. If instead we want to port stuff as we go, I agree and everything should be little `r`.